### PR TITLE
output(config) : ADD output.enabledLibraryTypes

### PR DESCRIPTION
**Pull request by jeffin143**
In reference with :

https://github.com/webpack/webpack/blob/d00e65529a4c5a4e91843f2d4178c66eff3784ae/schemas/WebpackOptions.json#L1663

https://github.com/webpack/webpack/blob/d00e65529a4c5a4e91843f2d4178c66eff3784ae/lib/WebpackOptionsApply.js#L224-L227

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].

### CODE REVIEW

File: src/content/configuration/output.md [hunk 1]
```diff
@@ -1085,6 +1085,21 @@ module.exports = {
 };
 ```
 
+## `output.enabledLibraryTypes`
+
+`array`
```
Comment (EugeneHlushko) [inline src/content/configuration/output.md:1090]: array of strings?

File: src/content/configuration/output.md [hunk 2]
```diff
@@ -1085,6 +1085,21 @@ module.exports = {
...
+
+List of library types enabled for use by entry points.
```
Comment (EugeneHlushko) [inline src/content/configuration/output.md:1092]: I think this needs some explanation, what happens when they require modules that arent in enabled types? What is the use case of the option

File: src/content/configuration/output.md (hunk 1 above)
Comment (jeffin143) [reply src/content/configuration/output.md:1090]: yes

How should , I add it
`array of strings` ?

File: src/content/configuration/output.md (hunk 2 above)
Comment (jeffin143) [reply src/content/configuration/output.md:1092]: I am not sure about the use case, but going through the code, it describes this as

If you are using a function as entry which sets "library", you need to add all potential library types to `output.enabledLibraryTypes`.

File: src/content/configuration/output.md (hunk 1 above)
Comment (snitin315) [reply src/content/configuration/output.md:1090]: [string]

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**